src/patternfly/components/TreeView/tree-view.scss

```
@@ -110,12 +111,16 @@ $pf-c-tree-view--MaxNesting: 10;
 }
 
 .pf-c-tree-view__node-toggle {
+  position: absolute;
+  top: var(--pf-c-tree-view__node--PaddingTop);
```

Can you create a var for this and set `--pf-c-tree-view__node-toggle--Top: var(--pf-c-tree-view__node--PaddingTop);`

```suggestion
  top: var(--pf-c-tree-view__node-toggle--Top);
```